backend/src/main/resources/db/migration/V1.16__rename_aux_table_columns.sql

```diff
@@ -0,0 +1,6 @@
+alter table sequence_upload_aux_table drop CONSTRAINT sequence_upload_aux_table_pkey;
+
+alter table sequence_upload_aux_table drop column segment_name;
```

Comment:
we should improve the migration - maybe create a new field sequence_submission_id which will be submission_id + segment_name and submission_id will become metadata_submission_id for all entries in the aux table at the time of migration